README.md

```diff
@@ -98,16 +105,26 @@ public final class QuickStart {
       Collections.singletonList(FRONTEND_KEY),
       Cumulative.create());
 
-  public static void main(String[] args) throws InterruptedException {
-    TagContextBuilder tagContextBuilder =
-      tagger.currentBuilder().put(FRONTEND_KEY, TagValue.create("mobile-ios9.3.5"));
-    viewManager.registerView(VIDEO_SIZE_VIEW);
-    // Process video.
-    // Record the processed video size.
-    try (Scope scopedTags = tagContextBuilder.buildScoped()) {
+  private static void processVideo() {
+    try (Scope scopedTags =
+           tagger
+             .currentBuilder()
+             .put(FRONTEND_KEY, TagValue.create("mobile-ios9.3.5")
+             .buildScoped()) {
+      // Processing video.
+      // ...
+
+      // Record the processed video size.
       statsRecorder.newMeasureMap().put(VIDEO_SIZE, 25648).record();
     }
+  }
+
+  public static void main(String[] args) throws InterruptedException {
+    viewManager.registerView(VIDEO_SIZE_VIEW);
+    processVideo();
```

Comment:
I still think that calling processVideo() directly from main() is slightly confusing, because it looks like processVideo() is only ever called once, instead of once per request.  It would help to either explain that it is a simplification or separate it from main, for example,

- Add an intermediate method call here, such as "handleRequests()", which is assumed to call processVideo, or
- Create three independent methods, such as "processVideo", "initialize" (registers view), and "printStats" (calls getView).

Reply:
Calling from main() here is kind of a simplification. I updated the example with the 2nd option, hopefully it makes it more clear that main() is like a simulated job.